python/pyspark/streaming/context.py

```diff
@@ -264,7 +280,9 @@ def checkpoint(self, directory):
         """
         self._jssc.checkpoint(directory)
 
-    def socketTextStream(self, hostname, port, storageLevel=StorageLevel.MEMORY_AND_DISK_2):
+    def socketTextStream(
+        self, hostname: str, port: int, storageLevel: StorageLevel = StorageLevel.MEMORY_AND_DISK_2
+    ) -> "DStream[str]":
```

Comment:
I think we can directly use `DStream[str]` here.

`"DStream[str]"` -> `DStream[str]` should be fine.

Reply:
Thanks for you reviewing!
We need to wait to inline type hints `python/pyspark/streaming/dstream.py` and have `DStream(Generic[T_co])` before use `DStream[str]` directly

Reply:
> We need to wait to inline type hints `python/pyspark/streaming/dstream.py` and have `DStream(Generic[T_co])` before use `DStream[str]` directly

Why do say that? `DStream` is already generic within stubs.

Reply:
To be precise, for `DStream[T]` you need `DStream` to be runtime generic. To avoid further rewrites, we can mark them as generics within this PR.